project/src/main/webapp/event-edit.html

```diff
@@ -55,6 +55,25 @@ <h3>Edit Your Event</h3>
               </div>
             </div>
             <br />
+            <div class="row">
+              <div class="col">
+                <div class="input-group">
+                  <div class="input-group-prepend">
+                    <span class="input-group-text" id="basic-addon1">Location</span>
+                  </div>
+                  <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Where your event will be held"
+                    id="location"
+                    name="location"
+                    required="required"
+                    aria-describedby="basic-addon1"
+                  />
+                </div>
+              </div>
+            </div>
+            <br />
             <div class="row">
```

Comment:
make sure to make 'description' 10 rows as well for consistentcy ( i saw that on create-event.html)

Reply:
Done